spec/models/user_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  user = User.find(1)
+
+  before(:each) { user.save }
+
+  it 'validates post name' do
+    user.name = nil
+    expect(user).to_not be_valid
+  end
+
+  it 'validates the presence of the posts_counter' do
+    user.posts_counter = nil
+    expect(user).to_not be_valid
+  end
+
+  it 'validates the numericality of the posts_counter' do
+    user.posts_counter = 'a'
+    expect(user).to_not be_valid
+  end
+
+  it 'validates the presence of the bio' do
+    user.bio = nil
+    expect(user).to_not be_valid
+  end
+
+  it 'validates the user photo' do
+    user.photo = nil
+    expect(user).to_not be_valid
+  end
+
+  describe '#recent_3_posts' do
+    before(:each) do
+      3.times do |i|
+        Post.new(title: "Post #{i}", text: "text#{i}", comments_counter: 0, likes_counter: 0, user_id: user.id)
+      end
+    end
+
+    it 'returns the last 3 posts' do
+      expect(user.recent_3_posts.length).to eq(3)
+    end
+  end
+end
```

Comment:
- [ ] Kindly fix the following issue to make the tests run successfully.

![image](https://user-images.githubusercontent.com/55635977/166426447-d991c9e8-2b53-492c-b82f-ddd55669d490.png)

Reply:
Kindly import seed.rb before running tests